custom_components/smart_envi/config_flow.py

```
@@ -0,0 +1,676 @@
+                    # Refresh device data
+                    coordinator_key = f"{DOMAIN}_coordinator_{self.config_entry.entry_id}"
+                    coordinator = domain_data.get(coordinator_key)
+                    if coordinator and hasattr(coordinator, "async_refresh_device"):
+                        if self._device_id in coordinator.device_ids:
+                            await coordinator.async_refresh_device(self._device_id)
+                    
+                    return self.async_create_entry(
+                        title="",
+                        data=self.config_entry.options or {},
+                    )
+            except Exception as e:
```

Ruff (0.14.8)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)